neqo-transport/src/connection/mod.rs

```diff
@@ -1137,6 +1173,37 @@ impl Connection {
         output
     }
 
+    // TODO: For the lack of a better name, `process_into` for now.
+    /// Process input and generate output.
+    #[must_use = "Output of the process function must be handled"]
+    pub fn process_into<'a>(
```

Comment:
This doesn't match your commentary either.  Did you forget to push something?

Reply:
In the PR description I stripped away the `TODO` comment. Otherwise it is the same:

```diff
     pub fn process_into<'a>(
         &mut self,
         input: Option<Datagram<&[u8]>>,
         now: Instant,
-        // TODO: Could this as well be an &mut [u8]?
         write_buffer: &'a mut Vec<u8>,
     ) -> Output<&'a [u8]> {
```

https://github.com/mxinden/neqo/blob/5d847eed43007e7a68cf66fa76c9728b8a6525e4/neqo-transport/src/connection/mod.rs#L1179-L1185

Note the rename to `process_into` is temporary. This is a large change. In order to incrementally compile I introduced some duplication.